tests/firstUppercase.spec.js

```diff
@@ -0,0 +1,13 @@
+describe('UserService', function () {
```

Comment:
I think this file can be removed? Instead, include empty files `gender.spec.js` and `removeAllVowels.spec.js`